src/components/ProductsTableRow/ProductsTableRow.tsx

```diff
@@ -1,38 +1,31 @@
 import './ProductsTableRow.scss';
-import { ProductDto } from 'dtos/ProductDto';
-import { useProducts } from 'hooks/useProducts';
 import { ReactElement } from 'react';
-import { toast } from 'react-toastify';
+import { ProductDto } from 'dtos/ProductDto';
+import { useCategories } from 'hooks/useCategories';
 
 interface Props {
   product: ProductDto;
 }
 
 export const ProductsTableRow = ({ product }: Props): ReactElement => {
-  const { removeProduct } = useProducts();
-
-  const handleDelete = (): void => {
-    removeProduct.mutate(product.id, {
-      onSuccess: () => {
-        toast.success(`Product "${product.name}" was succesfully removed.`);
-      },
-    });
-  };
-
+  const { categories } = useCategories();
+  const categoriesByProduct = [...new Set(categories.data?.filter((element) => product.categories?.includes(element.id)))].map(
+    (category) => category.name
+  );
   return (
-    <tr>
-      <td className="text-left">{product.id}</td>
-      <td className="text-left">{product.name}</td>
-      <td className="text-right products_table__hide_mobile">£{product.price.toFixed(2)}</td>
-      <td className="text-left products_table__hide_mobile">{product.description}</td>
-      {/*<td className="text-left products_table__hide_mobile">*/}
-      {/*  {product.categories?.map((category) => category.name).join(', ')}*/}
-      {/*</td>*/}
-      <td className="text-right">
-        <span className="products_table_row__delete" onClick={handleDelete} onKeyDown={handleDelete} role="button" tabIndex={0}>
-          Delete
-        </span>
-      </td>
-    </tr>
+    <>
+      <tr key={product.id}>
+        <td className="">{product.id}</td>
+        <td className="">{product.name}</td>
+        <td className="">£{product.price.toFixed(2)}</td>
+        <td className="">{product.description}</td>
+        <td className="">{categoriesByProduct.join(', ')}</td>
+        <td className="">
```

Comment:
should these classnames be empty?

Reply:
For now, as we move to tailwind and whilst we wait for the designs for the staff page